src/Core/src/Platform/iOS/MauiPageControl.cs

```diff
@@ -68,8 +68,10 @@ public void UpdateIndicatorSize()
 
 			float scale = (float)IndicatorSize / DefaultIndicatorSize;
 			var newTransform = CGAffineTransform.MakeScale(scale, scale);
-
-			Transform = newTransform;
+			foreach (var view in Subviews)
```

Comment:
Since `UpdateIndicatorSize()` is called from `LayoutSubviews()`, it could be called frequently during:
- Device rotation
- Keyboard appearance/dismissal
- Parent view layout changes

Could we detect if there are changes in the size?

```
double _lastAppliedIndicatorSize = -1;

if (Math.Abs(IndicatorSize - _lastAppliedIndicatorSize) < 0.001)
     return;
```

_NOTE: Can create a tolerance constant._

Reply:
@jsuarezruiz, I have updated the code section based on your suggestions.